app/assets/stylesheets/_vendor.scss

```diff
@@ -6,7 +6,6 @@
 @import 'basscss-sass/color-base';
 @import 'basscss-sass/color-forms';
 @import 'basscss-sass/color-tables';
-@import 'basscss-sass/type-scale';
```

Comment:
We'll need to bring this back if we've got some remaining usage of other `.h1-h6`classes.